src/main/routes/scenario.resource.js

```diff
@@ -2,49 +2,50 @@ import { ScenarioVerifier } from '../api-utilities/ScenarioVerifier';
 import { TemplateVerifier } from '../api-utilities/TemplateVerifier';
 import { ActorsOfActions } from '../api-utilities/SinglePropertyVerifier';
 import { Runner } from '../modules/Runner';
-import { QueryHandler } from '../db/queryHandler';
```

Comment:
Rubiana do you made this changes permanently? if you delete the import when we call use the request in the demo it will not work, please clarify tomorrow in a meeting what is the idea please

Reply:
As I said in the PR description, everything works the same. The implementation works the same.
I didn't delete the DB import, I changed places only. You can see that this import moved to the file src/main/server.js in line 1. 
But I don't mind clarifying this in a meeting, sure thing.

Reply:
It would be great stay tuned about this explaination as well.